vulnfeeds/cmd/cperepos/main.go

```diff
@@ -318,6 +332,7 @@ func MaybeGetSourceRepoFromDebian(mdir string, pkg string) string {
 func analyzeCPEDictionary(d CPEDict) (ProductToRepo VendorProductToRepoMap, DescriptionFrequency map[string]int) {
 	ProductToRepo = make(VendorProductToRepoMap)
 	DescriptionFrequency = make(map[string]int)
+	MaybeTryDebian := make(map[string]map[string]bool)
```

Comment:
Rather than do this complicated nesting, can we just create a struct to include both the Vendor and Product? We should be able to use that as a map key as well.

Reply:
Done in cb8158aec6e877cc7d19f6b8ce0d76466e9f2de3